pkg/kvstore/store/metrics.go

```diff
@@ -10,25 +10,29 @@ import (
 
 type Metrics struct {
 	KVStoreSyncQueueSize        metric.Vec[metric.Gauge]
+	KVStoreSyncErrors           metric.Vec[metric.Counter]
 	KVStoreInitialSyncCompleted metric.Vec[metric.Gauge]
 }
 
 func MetricsProvider() *Metrics {
 	return &Metrics{
 		KVStoreSyncQueueSize: metric.NewGaugeVec(metric.GaugeOpts{
-			ConfigName: metrics.Namespace + "_" + metrics.SubsystemKVStore + "_sync_queue_size",
-			Namespace:  metrics.Namespace,
-			Subsystem:  metrics.SubsystemKVStore,
-			Name:       "sync_queue_size",
-			Help:       "Number of elements queued for synchronization in the kvstore",
+			Namespace: metrics.Namespace,
+			Subsystem: metrics.SubsystemKVStore,
+			Name:      "sync_queue_size",
+			Help:      "Number of elements queued for synchronization in the kvstore",
+		}, []string{metrics.LabelScope, metrics.LabelSourceCluster}),
+		KVStoreSyncErrors: metric.NewCounterVec(metric.CounterOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: metrics.SubsystemKVStore,
+			Name:      "sync_errors_counter",
```

Comment:
```suggestion
			Name:      "sync_errors_total",
```

IIUC from https://prometheus.io/docs/practices/naming/